src/containerapp-preview/azext_containerapp_preview/_constants.py

```diff
@@ -10,3 +10,4 @@
 GA_CONTAINERAPP_EXTENSION_NAME = 'containerapp'
 MIN_GA_VERSION = '0.3.36'
 CONTAINERAPP_NAMESPACE = 'Microsoft.App'
+CONTAINER_APPS_RP = "Microsoft.App"
```

Comment:
could we use one var?

Reply:
Done. Use CONTAINER_APPS_RP and remove CONTAINERAPP_NAMESPACE .